roles/common/tasks/password-policy.yml

```diff
@@ -15,24 +17,47 @@
   with_nested:
     - [ /etc/pam.d/login, /etc/pam.d/passwd, /etc/pam.d/sshd, /etc/pam.d/su ]
     - [ "auth include common-auth", "account include common-account", "password include common-password" ]
-
-- name: set maximum password age
-  lineinfile: dest=/etc/login.defs regexp=^PASS_MAX_DAYS
-              line="PASS_MAX_DAYS   90" state=present
-
-- name: set minimum password age
-  lineinfile: dest=/etc/login.defs regexp=^PASS_MIN_DAYS
-              line="PASS_MIN_DAYS   1" state=present
-
+  when: os == 'ubuntu'
+  
 - name: prevent resuse of passwords
   lineinfile: dest=/etc/pam.d/common-password regexp=(pam_unix.so) state=present
               line="password [success=1 default=ignore] pam_unix.so obscure use_authtok sha512 remember=7 shadow"
               insertafter='^# here are the per-package modules'
               create=true
+  when: os == 'ubuntu'
 
 - name: implement password complexity
   lineinfile: dest=/etc/pam.d/common-password regexp=(pam_cracklib.so) state=present
               line="password requisite pam_cracklib.so ucredit=0 lcredit=-1 dcredit=-1 ocredit=0 retry=3 minlen=8 difok=3 reject_username"
               insertafter='^# here are the per-package modules'
               create=true
+  when: os == 'ubuntu'
```

Comment:
looks like a lot of `when: os == 'ubuntu'` in a row .. should we flip these into two bocks ... one for ubuntu one for rhel ?